### ISSUE TITLE
bug: spawn EPERM

### ISSUE THREAD

**Issue by gigaga** [simonhaenisch/md-to-pdf#55]
**Context:**

- version (`md-to-pdf -v`): 2.8.2
- platform (Unix, macOS, Windows): Windows
- node version: 10.15.3

**Describe the bug:**

After executing the command `md-to-pdf docs/README.md docs/output.pdf`, I get the output:

>  × generating PDF from docs/README.md
>    → spawn EPERM

Thanks for your help,

**Comment by simonhaenisch** [comment]
Probably you need to run cmd.exe as an administrator so that the terminal has the permission to write files?

**Comment by gigaga** [comment]
Hello,

In fact, It does not work even from CMD executed as Administrator :
![image](https://user-images.githubusercontent.com/6095992/72079312-f6ad8080-32fa-11ea-9536-da34d03f7ea2.png)

**Comment by simonhaenisch** [comment]
Ok... can you please add the `--debug` flag so that the full error message is printed?

**Comment by gigaga** [comment]
Hello @simonhaenisch,

Of course, below the related output.
![image](https://user-images.githubusercontent.com/6095992/72140749-bc42f280-3391-11ea-8394-d3a8b2c30353.png)

**Comment by simonhaenisch** [comment]
After searching around for a little bit, there's multiple reasons that could cause this, but as I already suspected, there's a permission error from the system for spawning the child process (that launches puppeteer).

The most common cause seem an anti-virus software interfering by blocking the process from spawning (see e. g. https://stackoverflow.com/q/57922756/2897426 and https://github.com/electron-userland/electron-builder/issues/3985).

Closing, as this seems unrelated to `md-to-pdf` itself. Feel free to keep commenting and let us know if you find a solution.
